.golangci.yml

```diff
@@ -29,6 +29,7 @@ linters:
     - depguard
     - dogsled
     - errcheck
+    - errorlint # errorlint is a linter for that can be used to find code that will cause problems with the error wrapping scheme introduced in Go 1.13. [fast: false, auto-fix: false]
```

Comment:
```suggestion
    - errorlint # errorlint is a linter that can be used to find code that will cause problems with the error wrapping scheme introduced in Go 1.13. [fast: false, auto-fix: false]
```

Reply:
this is actually a copy/paste from `golangci-lint linters` do you want to open a PR there?